### PR TITLE
Add initial roles

### PR DESCRIPTION
**Pull request by machacekondra**
This patch add initial implementation of initial roles.

### CODE REVIEW

**Comment by machacekondra** [comment]
@mwperina

File: examples/library/parted.py [hunk 1]
```diff
@@ -0,0 +1,705 @@
+#!/usr/bin/python
```
Comment (mwperina) [inline examples/library/parted.py]: Not needed now, please remove the file

File: examples/openshift.yml [hunk 2]
```diff
@@ -0,0 +1,185 @@
+---
```
Comment (mwperina) [inline examples/openshift.yml]: Not needed now, please remove this file

File: examples/openshift_vars.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (mwperina) [inline examples/openshift_vars.yml]: Not needed now, please remove the file

File: examples/ovirt_cluster_upgrade.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (mwperina) [inline examples/ovirt_cluster_upgrade.yml]: Not needed now, please remove the file

File: examples/ovirt_image_template.yml [hunk 5]
```diff
@@ -0,0 +1,22 @@
+---
+- name: oVirt image template
+  hosts: localhost
+  connection: local
+  gather_facts: false
+
+  vars:
+    engine_url: https://ondra.local/ovirt-engine/api
```
Comment (mwperina) [inline examples/ovirt_image_template.yml]: engine_url: https://ovirt-engine.example.com/ovirt-engine/api

File: examples/ovirt_infra_vars.yml [hunk 6]
```diff
@@ -0,0 +1,217 @@
+---
+###########################
+# REST API variables
+###########################
+engine_url: https://ondra.local/ovirt-engine/api
+#engine_url: https://ovirt-engine.example.com/ovirt-engine/api
```
Comment (mwperina) [inline examples/ovirt_infra_vars.yml]: Please use this one and remove ondra.local

File: examples/ovirt_infra_vars.yml [hunk 7]
```diff
@@ -0,0 +1,217 @@
...
+engine_user: admin@internal
+engine_password: 123456
+#engine_cafile: /etc/pki/ovirt-engine/ca.pem
+engine_insecure: true
```
Comment (mwperina) [inline examples/ovirt_infra_vars.yml]: Please use CA instead of insecure access

File: README.md [hunk 8]
```diff
@@ -1,2 +1,39 @@
-# ovirt-ansible
-This repository holds all oVirt related Ansible roles
+[![Build Status](https://travis-ci.org/machacekondra/ovirt-ansible-roles.svg?branch=master)](https://travis-ci.org/machacekondra/ovirt-ansible-roles)
```
Comment (mwperina) [inline README.md]: Is this still valid for official oVirt repo?

File: README.md [hunk 9]
```diff
@@ -1,2 +1,39 @@
...
+[ovirt-infra]: https://github.com/machacekondra/ovirt-ansible-roles/blob/master/roles/ovirt-infra/README.md
+[ovirt-image-template]: https://github.com/machacekondra/ovirt-ansible-roles/blob/master/roles/ovirt-image-template/README.md
+[ovirt-vm-infra]: https://github.com/machacekondra/ovirt-ansible-roles/blob/master/roles/ovirt-vm-infra/README.md
+[ovirt-aaa-jdbc]: https://github.com/machacekondra/ovirt-ansible-roles/blob/master/roles/ovirt-aaa-jdbc/README.md
+[ovirt-clusters]: https://github.com/machacekondra/ovirt-ansible-roles/blob/master/roles/ovirt-clusters/README.md
+[ovirt-datacenters]: https://github.com/machacekondra/ovirt-ansible-roles/blob/master/roles/ovirt-datacenters/README.md
+[ovirt-hosts]: https://github.com/machacekondra/ovirt-ansible-roles/blob/master/roles/ovirt-hosts/README.md
+[ovirt-networks]: https://github.com/machacekondra/ovirt-ansible-roles/blob/master/roles/ovirt-networks/README.md
+[ovirt-permissions]: https://github.com/machacekondra/ovirt-ansible-roles/blob/master/roles/ovirt-permissions/README.md
+[ovirt-storages]: https://github.com/machacekondra/ovirt-ansible-roles/blob/master/roles/ovirt-storages/README.md
```
Comment (mwperina) [inline README.md]: Shouldn't above links be replaced with https://github.com/oVirt/ovirt-ansible

File: roles/ovirt-image-template/meta/main.yml [hunk 10]
```diff
@@ -0,0 +1,20 @@
+galaxy_info:
+  author: Ondra Machacek
+  description: Role to deploy Manage IQ into oVirt.
```
Comment (mwperina) [inline roles/ovirt-image-template/meta/main.yml]: Please fix

File: roles/ovirt-image-template/meta/main.yml [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+  company: Red Hat, Inc.
+
+  license: license (GPLv2, CC-BY, etc)
```
Comment (mwperina) [inline roles/ovirt-image-template/meta/main.yml]: Apache License 2.0

File: roles/ovirt-image-template/meta/main.yml [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+
+  min_ansible_version: 2.2
```
Comment (mwperina) [inline roles/ovirt-image-template/meta/main.yml]: Ansible 2.3

File: roles/ovirt-image-template/meta/main.yml [hunk 13]
```diff
@@ -0,0 +1,20 @@
...
+
+  platforms:
+  - name: EL
+    versions:
+    - 7
+  - name: Fedora
+    versions:
+    - 24
+
+  galaxy_tags: [manageiq, cloudforms, ovirt, rhv, rhev, virtualization]
```
Comment (mwperina) [inline roles/ovirt-image-template/meta/main.yml]: Please remove manageiq/cloudforms reference

File: examples/ovirt_image_template.yml [hunk 14]
```diff
@@ -0,0 +1,22 @@
...
+    engine_user: admin@internal
+    engine_password: 123456
+    engine_cafile: /home/omachace/Downloads/ca.pem
```
Comment (mwperina) [inline examples/ovirt_image_template.yml]: engine_cafile: /etc/pki/ovirt-engine/ca.pem

File: examples/ovirt_image_template.yml [hunk 15]
```diff
@@ -0,0 +1,22 @@
+---
+- name: oVirt image template
+  hosts: localhost
+  connection: local
+  gather_facts: false
+
+  vars:
+    engine_url: https://ondra.local/ovirt-engine/api
+    engine_user: admin@internal
+    engine_password: 123456
```
Comment (mwperina) [inline examples/ovirt_image_template.yml]: Use vault instead?

File: examples/ovirt_infra_vars.yml [hunk 16]
```diff
@@ -0,0 +1,217 @@
+---
+###########################
+# REST API variables
+###########################
+engine_url: https://ondra.local/ovirt-engine/api
+#engine_url: https://ovirt-engine.example.com/ovirt-engine/api
+engine_user: admin@internal
+engine_password: 123456
```
Comment (mwperina) [inline examples/ovirt_infra_vars.yml]: Use vault instead?

File: examples/ovirt_infra_vars.yml [hunk 17]
```diff
@@ -0,0 +1,217 @@
...
+template_name: rhel7
+template_memory: 1GiB
+template_cpu: 1
+template_disk_size: 20GiB
+template_nics:
+ - name: nic1
+   profile_name: ovirtmgmt
+   interface: virtio
+
+
```
Comment (mwperina) [inline examples/ovirt_infra_vars.yml]: Aren't below VMs OpenShift related?

File: roles/ovirt-aaa-jdbc/meta/main.yml [hunk 18]
```diff
@@ -0,0 +1,222 @@
+galaxy_info:
+  author: your name
+  description: your description
+  company: your company (optional)
```
Comment (mwperina) [inline roles/ovirt-aaa-jdbc/meta/main.yml]: Please fix above

File: roles/ovirt-aaa-jdbc/meta/main.yml [hunk 19]
```diff
@@ -0,0 +1,222 @@
...
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
```
Comment (mwperina) [inline roles/ovirt-aaa-jdbc/meta/main.yml]: Apache License 2.0

File: roles/ovirt-aaa-jdbc/meta/main.yml [hunk 20]
```diff
@@ -0,0 +1,222 @@
...
+  #  - all
+  #  - 10
+  #  - 11.0
+  #  - 11.1
+  #  - 11.2
+  #  - 11.3
+  #- name: eos
+  #  versions:
+  #  - all
+  #  - Any
```
Comment (mwperina) [inline roles/ovirt-aaa-jdbc/meta/main.yml]: Do we need above huge comment included?

File: roles/ovirt-clusters/library/ovirt_clusters.py [hunk 21]
```diff
@@ -0,0 +1,575 @@
+#!/usr/bin/python
```
Comment (mwperina) [inline roles/ovirt-clusters/library/ovirt_clusters.py:1]: Isn't this module part of Ansible 2.3?

File: roles/ovirt-clusters/meta/main.yml [hunk 22]
```diff
@@ -0,0 +1,222 @@
+galaxy_info:
+  author: your name
+  description: your description
+  company: your company (optional)
```
Comment (mwperina) [inline roles/ovirt-clusters/meta/main.yml]: Please fix above

File: roles/ovirt-clusters/meta/main.yml [hunk 23]
```diff
@@ -0,0 +1,222 @@
...
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
```
Comment (mwperina) [inline roles/ovirt-clusters/meta/main.yml]: Apache License 2.0

File: roles/ovirt-clusters/meta/main.yml [hunk 24]
```diff
@@ -0,0 +1,222 @@
...
+  #  - all
+  #  - 10
+  #  - 11.0
+  #  - 11.1
+  #  - 11.2
+  #  - 11.3
+  #- name: eos
+  #  versions:
+  #  - all
+  #  - Any
```
Comment (mwperina) [inline roles/ovirt-clusters/meta/main.yml]: Do we need huge comment above?

File: roles/ovirt-datacenters/meta/main.yml [hunk 25]
```diff
@@ -0,0 +1,222 @@
+galaxy_info:
+  author: your name
+  description: your description
+  company: your company (optional)
```
Comment (mwperina) [inline roles/ovirt-datacenters/meta/main.yml]: Please fix above

File: roles/ovirt-datacenters/meta/main.yml [hunk 26]
```diff
@@ -0,0 +1,222 @@
...
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
```
Comment (mwperina) [inline roles/ovirt-datacenters/meta/main.yml]: Apache License 2.0

File: roles/ovirt-datacenters/meta/main.yml [hunk 27]
```diff
@@ -0,0 +1,222 @@
...
+  #  - all
+  #  - 10
+  #  - 11.0
+  #  - 11.1
+  #  - 11.2
+  #  - 11.3
+  #- name: eos
+  #  versions:
+  #  - all
+  #  - Any
```
Comment (mwperina) [inline roles/ovirt-datacenters/meta/main.yml]: Do we need that huge comment above?

File: roles/ovirt-hosts/meta/main.yml [hunk 28]
```diff
@@ -0,0 +1,222 @@
+galaxy_info:
+  author: your name
+  description: your description
+  company: your company (optional)
```
Comment (mwperina) [inline roles/ovirt-hosts/meta/main.yml]: Please fix above

File: roles/ovirt-hosts/meta/main.yml [hunk 29]
```diff
@@ -0,0 +1,222 @@
...
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
```
Comment (mwperina) [inline roles/ovirt-hosts/meta/main.yml]: Apache License 2.0

File: roles/ovirt-hosts/meta/main.yml [hunk 30]
```diff
@@ -0,0 +1,222 @@
...
+  #  - all
+  #  - 10
+  #  - 11.0
+  #  - 11.1
+  #  - 11.2
+  #  - 11.3
+  #- name: eos
+  #  versions:
+  #  - all
+  #  - Any
```
Comment (mwperina) [inline roles/ovirt-hosts/meta/main.yml]: Huge comment?

File: roles/ovirt-infra/README.md [hunk 31]
```diff
@@ -0,0 +1,213 @@
...
+
+Apache License 2.0
+
+[ovirt-aaa-jdbc]: https://github.com/machacekondra/ovirt-ansible-roles/blob/master/roles/ovirt-aaa-jdbc/README.md
+[ovirt-clusters]: https://github.com/machacekondra/ovirt-ansible-roles/blob/master/roles/ovirt-clusters/README.md
+[ovirt-datacenters]: https://github.com/machacekondra/ovirt-ansible-roles/blob/master/roles/ovirt-datacenters/README.md
+[ovirt-hosts]: https://github.com/machacekondra/ovirt-ansible-roles/blob/master/roles/ovirt-hosts/README.md
+[ovirt-networks]: https://github.com/machacekondra/ovirt-ansible-roles/blob/master/roles/ovirt-networks/README.md
+[ovirt-permissions]: https://github.com/machacekondra/ovirt-ansible-roles/blob/master/roles/ovirt-permissions/README.md
+[ovirt-storages]: https://github.com/machacekondra/ovirt-ansible-roles/blob/master/roles/ovirt-storages/README.md
```
Comment (mwperina) [inline roles/ovirt-infra/README.md]: Shouldn't above links be replaced with https://github.com/oVirt/ovirt-ansible ?

File: roles/ovirt-infra/meta/main.yml [hunk 32]
```diff
@@ -0,0 +1,226 @@
+galaxy_info:
+  author: your name
+  description: your description
+  company: your company (optional)
```
Comment (mwperina) [inline roles/ovirt-infra/meta/main.yml]: Please fix above

File: roles/ovirt-infra/meta/main.yml [hunk 33]
```diff
@@ -0,0 +1,226 @@
...
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
```
Comment (mwperina) [inline roles/ovirt-infra/meta/main.yml]: Apache License 2.0

File: roles/ovirt-infra/meta/main.yml [hunk 34]
```diff
@@ -0,0 +1,226 @@
...
+  #  - all
+  #  - 10
+  #  - 11.0
+  #  - 11.1
+  #  - 11.2
+  #  - 11.3
+  #- name: eos
+  #  versions:
+  #  - all
+  #  - Any
```
Comment (mwperina) [inline roles/ovirt-infra/meta/main.yml]: Do we need the huge comment above?

File: roles/ovirt-networks/meta/main.yml [hunk 35]
```diff
@@ -0,0 +1,222 @@
+galaxy_info:
+  author: your name
+  description: your description
+  company: your company (optional)
```
Comment (mwperina) [inline roles/ovirt-networks/meta/main.yml]: Please fix above

File: roles/ovirt-networks/meta/main.yml [hunk 36]
```diff
@@ -0,0 +1,222 @@
...
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
```
Comment (mwperina) [inline roles/ovirt-networks/meta/main.yml]: Apache License 2.0

File: roles/ovirt-networks/meta/main.yml [hunk 37]
```diff
@@ -0,0 +1,222 @@
...
+  #  - all
+  #  - 10
+  #  - 11.0
+  #  - 11.1
+  #  - 11.2
+  #  - 11.3
+  #- name: eos
+  #  versions:
+  #  - all
+  #  - Any
```
Comment (mwperina) [inline roles/ovirt-networks/meta/main.yml]: Do we need a huge comment above?

File: roles/ovirt-permissions/meta/main.yml [hunk 38]
```diff
@@ -0,0 +1,222 @@
+galaxy_info:
+  author: your name
+  description: your description
+  company: your company (optional)
```
Comment (mwperina) [inline roles/ovirt-permissions/meta/main.yml]: Please fix above

File: roles/ovirt-permissions/meta/main.yml [hunk 39]
```diff
@@ -0,0 +1,222 @@
...
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
```
Comment (mwperina) [inline roles/ovirt-permissions/meta/main.yml]: Apache License 2.0

File: roles/ovirt-permissions/meta/main.yml [hunk 40]
```diff
@@ -0,0 +1,222 @@
...
+  #  - all
+  #  - 10
+  #  - 11.0
+  #  - 11.1
+  #  - 11.2
+  #  - 11.3
+  #- name: eos
+  #  versions:
+  #  - all
+  #  - Any
```
Comment (mwperina) [inline roles/ovirt-permissions/meta/main.yml]: Do we need the huge comment above?

File: roles/ovirt-storages/library/ovirt_storage_connections.py [hunk 41]
```diff
@@ -0,0 +1,282 @@
+#!/usr/bin/python
```
Comment (mwperina) [inline roles/ovirt-storages/library/ovirt_storage_connections.py:1]: Isn't the module part of Ansible 2.3?

File: roles/ovirt-storages/meta/main.yml [hunk 42]
```diff
@@ -0,0 +1,222 @@
+galaxy_info:
+  author: your name
+  description: your description
+  company: your company (optional)
```
Comment (mwperina) [inline roles/ovirt-storages/meta/main.yml]: Please fix above

File: roles/ovirt-storages/meta/main.yml [hunk 43]
```diff
@@ -0,0 +1,222 @@
...
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
```
Comment (mwperina) [inline roles/ovirt-storages/meta/main.yml]: Apache License 2.0

File: roles/ovirt-storages/meta/main.yml [hunk 44]
```diff
@@ -0,0 +1,222 @@
...
+  #  - all
+  #  - 10
+  #  - 11.0
+  #  - 11.1
+  #  - 11.2
+  #  - 11.3
+  #- name: eos
+  #  versions:
+  #  - all
+  #  - Any
```
Comment (mwperina) [inline roles/ovirt-storages/meta/main.yml]: Do we need that huge comment above?

File: roles/ovirt-vm-infra/meta/main.yml [hunk 45]
```diff
@@ -0,0 +1,222 @@
+galaxy_info:
+  author: your name
+  description: your description
+  company: your company (optional)
```
Comment (mwperina) [inline roles/ovirt-vm-infra/meta/main.yml]: Please fix above

File: roles/ovirt-vm-infra/meta/main.yml [hunk 46]
```diff
@@ -0,0 +1,222 @@
...
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
```
Comment (mwperina) [inline roles/ovirt-vm-infra/meta/main.yml]: Apache License 2.0

File: roles/ovirt-vm-infra/meta/main.yml [hunk 47]
```diff
@@ -0,0 +1,222 @@
...
+  #  - all
+  #  - 10
+  #  - 11.0
+  #  - 11.1
+  #  - 11.2
+  #  - 11.3
+  #- name: eos
+  #  versions:
+  #  - all
+  #  - Any
```
Comment (mwperina) [inline roles/ovirt-vm-infra/meta/main.yml]: Do we need that huge comment above?

File: roles/ovirt-aaa-jdbc/meta/main.yml [hunk 48]
```diff
@@ -0,0 +1,222 @@
...
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
+
+  min_ansible_version: 1.2
```
Comment (mwperina) [inline roles/ovirt-aaa-jdbc/meta/main.yml]: 2.3?

File: roles/ovirt-clusters/meta/main.yml [hunk 49]
```diff
@@ -0,0 +1,222 @@
...
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
+
+  min_ansible_version: 1.2
```
Comment (mwperina) [inline roles/ovirt-clusters/meta/main.yml]: 2.3?

File: roles/ovirt-datacenters/meta/main.yml [hunk 50]
```diff
@@ -0,0 +1,222 @@
...
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
+
+  min_ansible_version: 1.2
```
Comment (mwperina) [inline roles/ovirt-datacenters/meta/main.yml]: 2.3?

File: roles/ovirt-hosts/meta/main.yml [hunk 51]
```diff
@@ -0,0 +1,222 @@
...
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
+
+  min_ansible_version: 1.2
```
Comment (mwperina) [inline roles/ovirt-hosts/meta/main.yml]: 2.3?

File: roles/ovirt-infra/meta/main.yml [hunk 52]
```diff
@@ -0,0 +1,226 @@
...
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
+
+  min_ansible_version: 1.2
```
Comment (mwperina) [inline roles/ovirt-infra/meta/main.yml]: 2.3?

File: roles/ovirt-networks/meta/main.yml [hunk 53]
```diff
@@ -0,0 +1,222 @@
...
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
+
+  min_ansible_version: 1.2
```
Comment (mwperina) [inline roles/ovirt-networks/meta/main.yml]: 2.3?

File: roles/ovirt-permissions/meta/main.yml [hunk 54]
```diff
@@ -0,0 +1,222 @@
...
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
+
+  min_ansible_version: 1.2
```
Comment (mwperina) [inline roles/ovirt-permissions/meta/main.yml]: 2.3?

File: roles/ovirt-storages/meta/main.yml [hunk 55]
```diff
@@ -0,0 +1,222 @@
...
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
+
+  min_ansible_version: 1.2
```
Comment (mwperina) [inline roles/ovirt-storages/meta/main.yml]: 2.3?

File: roles/ovirt-vm-infra/meta/main.yml [hunk 56]
```diff
@@ -0,0 +1,222 @@
...
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
+
+  min_ansible_version: 1.2
```
Comment (mwperina) [inline roles/ovirt-vm-infra/meta/main.yml]: 2.3?

File: roles/ovirt-aaa-jdbc/meta/main.yml [hunk 57]
```diff
@@ -0,0 +1,222 @@
...
+  #  - 11.0
+  #  - 11.1
+  #  - 11.2
+  #  - 11.3
+  #- name: eos
+  #  versions:
+  #  - all
+  #  - Any
+
+  galaxy_tags: []
```
Comment (mwperina) [inline roles/ovirt-aaa-jdbc/meta/main.yml]: ovirt rhv rhev virtualization ?

File: roles/ovirt-clusters/meta/main.yml [hunk 58]
```diff
@@ -0,0 +1,222 @@
...
+  #  - 11.0
+  #  - 11.1
+  #  - 11.2
+  #  - 11.3
+  #- name: eos
+  #  versions:
+  #  - all
+  #  - Any
+
+  galaxy_tags: []
```
Comment (mwperina) [inline roles/ovirt-clusters/meta/main.yml]: ovirt rhv rhev virtualization ?

File: roles/ovirt-datacenters/meta/main.yml [hunk 59]
```diff
@@ -0,0 +1,222 @@
...
+  #  - 11.0
+  #  - 11.1
+  #  - 11.2
+  #  - 11.3
+  #- name: eos
+  #  versions:
+  #  - all
+  #  - Any
+
+  galaxy_tags: []
```
Comment (mwperina) [inline roles/ovirt-datacenters/meta/main.yml]: ovirt rhv rhev virtualization ?

File: roles/ovirt-hosts/meta/main.yml [hunk 60]
```diff
@@ -0,0 +1,222 @@
...
+  #  - 11.0
+  #  - 11.1
+  #  - 11.2
+  #  - 11.3
+  #- name: eos
+  #  versions:
+  #  - all
+  #  - Any
+
+  galaxy_tags: []
```
Comment (mwperina) [inline roles/ovirt-hosts/meta/main.yml]: ovirt rhv rhev virtualization ?

File: roles/ovirt-infra/meta/main.yml [hunk 61]
```diff
@@ -0,0 +1,226 @@
...
+  #  - 11.0
+  #  - 11.1
+  #  - 11.2
+  #  - 11.3
+  #- name: eos
+  #  versions:
+  #  - all
+  #  - Any
+
+  galaxy_tags: []
```
Comment (mwperina) [inline roles/ovirt-infra/meta/main.yml]: ovirt rhv rhev virtualization ?

File: roles/ovirt-networks/meta/main.yml [hunk 62]
```diff
@@ -0,0 +1,222 @@
...
+  #  - 11.0
+  #  - 11.1
+  #  - 11.2
+  #  - 11.3
+  #- name: eos
+  #  versions:
+  #  - all
+  #  - Any
+
+  galaxy_tags: []
```
Comment (mwperina) [inline roles/ovirt-networks/meta/main.yml]: ovirt rhv rhev virtualization ?

File: roles/ovirt-permissions/meta/main.yml [hunk 63]
```diff
@@ -0,0 +1,222 @@
...
+  #  - 11.0
+  #  - 11.1
+  #  - 11.2
+  #  - 11.3
+  #- name: eos
+  #  versions:
+  #  - all
+  #  - Any
+
+  galaxy_tags: []
```
Comment (mwperina) [inline roles/ovirt-permissions/meta/main.yml]: ovirt rhv rhev virtualization ?

File: roles/ovirt-storages/meta/main.yml [hunk 64]
```diff
@@ -0,0 +1,222 @@
...
+  #  - 11.0
+  #  - 11.1
+  #  - 11.2
+  #  - 11.3
+  #- name: eos
+  #  versions:
+  #  - all
+  #  - Any
+
+  galaxy_tags: []
```
Comment (mwperina) [inline roles/ovirt-storages/meta/main.yml]: ovirt rhv rhev virtualization ?

File: roles/ovirt-vm-infra/meta/main.yml [hunk 65]
```diff
@@ -0,0 +1,222 @@
...
+  #  - 11.0
+  #  - 11.1
+  #  - 11.2
+  #  - 11.3
+  #- name: eos
+  #  versions:
+  #  - all
+  #  - Any
+
+  galaxy_tags: []
```
Comment (mwperina) [inline roles/ovirt-vm-infra/meta/main.yml]: ovirt rhv rhev virtualization ?

**Review by mwperina — CHANGES_REQUESTED** [review]

File: roles/ovirt-clusters/library/ovirt_clusters.py (hunk 21 above)
Comment (machacekondra) [reply roles/ovirt-clusters/library/ovirt_clusters.py:1]: Yes, but this is patched with `post_copy` policy.

File: roles/ovirt-storages/library/ovirt_storage_connections.py (hunk 41 above)
Comment (machacekondra) [reply roles/ovirt-storages/library/ovirt_storage_connections.py:1]: Unfortunatelly , no:

 https://github.com/ansible/ansible/pull/22239

File: examples/ovirt_vm_infra.yml [hunk 66]
```diff
@@ -0,0 +1,54 @@
+---
+- name: oVirt infra
+  hosts: localhost
+  connection: local
+  gather_facts: false
+
+  vars_files:
+    # Contains encrypted `engine_password` varibale using ansible-vault
+    - passwords.yml
```
Comment (mgoldboi) [inline examples/ovirt_vm_infra.yml:9]: do we plan to include this file template on the examples as well?

File: examples/ovirt_vm_infra.yml [hunk 67]
```diff
@@ -0,0 +1,54 @@
...
+    engine_user: admin@internal
+    engine_cafile: /etc/pki/ovirt-engine/ca.pem
+
+    httpd_vm:
+      cluster: production
+      domain: example.com
+      template: rhel7
+      memory: 2GiB
+      cores: 2
+      ssh_key: ssh-rsa AAA...LGx user@fqdn
```
Comment (mgoldboi) [inline examples/ovirt_vm_infra.yml:22]: should the passowrds.yml file includes cloud-init passwords for those vms, or should we list those here, I think those would be probably needed.

**Review by mwperina — APPROVED** [review]
👍

**Comment by mwperina** [comment]
@mykaul @mgoldboi

Are you OK with merging this?

File: examples/ovirt_infra_vars.yml [hunk 68]
```diff
@@ -0,0 +1,126 @@
+---
+###########################
+# REST API variables
+###########################
+engine_url: https://ovirt-engine.example.com/ovirt-engine/api
```
Comment (mgoldboi) [inline examples/ovirt_infra_vars.yml:5]: should we change the url to include only FQDN? aren't the suffixes always the same (/ovirt-engine/api)?

File: roles/ovirt-vm-infra/README.md [hunk 69]
```diff
@@ -0,0 +1,108 @@
...
+- name: oVirt infra
+  hosts: localhost
+  connection: local
+  gather_facts: false
+
+  vars:
+    engine_url: https://ovirt-engine.example.com/ovirt-engine/api
+    engine_user: admin@internal
+    engine_password: 123456
+    engine_cafile: /home/omachace/Downloads/ca.pem
```
Comment (mgoldboi) [inline roles/ovirt-vm-infra/README.md]: should be changed to /etc..?

File: examples/ovirt_infra_vars.yml [hunk 70]
```diff
@@ -0,0 +1,126 @@
...
+   profile: production
+
+###########################
+# Hosts
+###########################
+hosts:
+ - name: myhost
+   address: 1.2.3.5
+   cluster: production
+   password: 123456
```
Comment (mgoldboi) [inline examples/ovirt_infra_vars.yml:32]: should we store those on the password.yml as well?

**Review by mgoldboi — APPROVED** [review]
few comments, no blockers. Thanks!

**Comment by mgoldboi** [comment]
@machacekondra can we merge the changes before the webinar tomorrow?

**Comment by machacekondra** [comment]
Sure, will do.

File: examples/ovirt_vm_infra.yml (hunk 66 above)
Comment (machacekondra) [reply examples/ovirt_vm_infra.yml:9]: done

File: roles/ovirt-vm-infra/README.md (hunk 69 above)
Comment (machacekondra) [reply roles/ovirt-vm-infra/README.md]: done

File: examples/ovirt_vm_infra.yml (hunk 67 above)
Comment (machacekondra) [reply examples/ovirt_vm_infra.yml:22]: I will change in another PR

File: examples/ovirt_infra_vars.yml (hunk 68 above)
Comment (machacekondra) [reply examples/ovirt_infra_vars.yml:5]: I will change in another PR

File: examples/ovirt_infra_vars.yml (hunk 70 above)
Comment (machacekondra) [reply examples/ovirt_infra_vars.yml:32]: I will change in another PR, consolidating all passwords

**Pull request merged**